pytest.ini

```
@@ -0,0 +1,4 @@
+[pytest]
+addopts = --ignore=pic_sleuth/config/*
```

We've ignored using `coverage:run` instead.

Change to this to avoid specifying the --cov option when running from the terminal. You can now simply run `pytest`.

```suggestion
addopts = --cov=pic_sleuth --durations=25 -n auto -v
```